azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/ErrorInfoView.kt

```diff
@@ -45,32 +48,36 @@ internal class ErrorInfoView(private val rootView: View) {
     }
 
     private fun displaySnackBar(
-        it: CallStateError,
+        error: CallStateError,
         appLocalizationProvider: LocalizationProvider
     ) {
-        val errorMessage = getErrorMessage(it, appLocalizationProvider)
+        val errorMessage = getErrorMessage(error, appLocalizationProvider)
 
-        if (errorMessage.isNotEmpty()) {
-            snackBarTextView.text = errorMessage
-            if (!snackBar.isShown) {
-                snackBar.show()
+        if (errorMessage.isBlank()) return
+        snackBarTextView.text = errorMessage
+        snackBar.run {
+            if (isShown) {
```

Comment:
Just to be sure here, if the snackBar was already shown before, why are we dismissing it and showing again? To load the new errorMessage? Or it was a typo?

Reply:
@Mohtasim yes, that is correct.  If there is a snackBar already showing and we have a new non-blank message incoming, we dismiss the snackBar and show it again with the new message.

Reply:
Thanks.